package/Dockerfile

```diff
@@ -0,0 +1,75 @@
+
+#!BuildTag: yast-cpp
+
+FROM opensuse/tumbleweed
+
+# do not install the files marked as documentation (use "rpm --excludedocs")
+RUN sed -i -e "s/^.*rpm.install.excludedocs.*/rpm.install.excludedocs = yes/" /etc/zypp/zypp.conf
+
+# import the YaST OBS GPG key
+COPY YaST:Head.pub /usr/share/gpg-keys/
+RUN rpm --import /usr/share/gpg-keys/YaST:Head.pub
+
+# Prefer the packages from the YaST:Head repository
+# TODO -p 50 does not work, but should be usless anyway, just like --refresh
+RUN zypper addrepo --refresh http://download.opensuse.org/repositories/YaST:/Head/openSUSE_Tumbleweed yast
+
+RUN zypper --non-interactive install --no-recommends \
+  aspell-en \
+  autoconf \
+  automake \
+  bison \
+  boost-devel \
+  brp-check-suse \
+  brp-extract-appdata \
+  build \
+  cmake \
+  dejagnu \
+  docbook-xsl-stylesheets \
+  doxygen \
+  fdupes \
+  flex \
+  gcc-c++ \
+  git \
+  grep \
+  hwinfo-devel \
+  libtool \
+  libxslt \
+  obs-service-source_validator \
+  openSUSE-release-ftp \
+  rpm-build \
+  ruby-devel \
+  "rubygem(coveralls)" \
+  "rubygem(fast_gettext)" \
+  "rubygem(gettext)" \
+  "rubygem(raspell)" \
+  "rubygem(rspec)" \
+  "rubygem(rubocop)" \
+  "rubygem(simplecov)" \
+  "rubygem(yard)" \
+  "rubygem(yast-rake)" \
+  screen \
+  sgml-skel \
+  ShellCheck \
+  update-desktop-files \
+  which \
+  yast2-core-devel \
+  yast2-devtools \
+  yast2-devtools \
```

Comment:
yast2-devtools listed twice

Reply:
probably was already in the original `Dockerfile`, I just sorted the packages alphabetically.